### ISSUE TITLE
Enhance main application url redirect 

### ISSUE THREAD

**Issue by gmella** [eXist-db/eXide#46]
Hi, 

I propose to consider an additional empty path to be redirected to the main index.html page into templates/existdb/controller.xql

May do the next line do the trick :

```
if ($exist:path = ("/","")) then
```

in replacement of first test that is:

```
if ($exist:path eq "/") then
```

This would help a little bit more useability and confidentiality.
## Thanks a lot for your work!

Guillaume

**Comment by dizzzz** [comment]
Hi Guillaume,

interesting suggestion; what does it exactly solve? Can you provide an URL that does not work right now, which will work with the fix?

regards  Dannes

**Comment by gmella** [comment]
It will just prevent to show the listing of our application roots. 
For example this could hide the file stanford-ner.iml or any other one listed from http://www.exist-db.org/exist/apps/stanford-ner

**Comment by dizzzz** [comment]
Thnx that is very useful information

**Comment by wolfgangmm** [comment]
Thanks for the hint. Should be fixed in aabf155b230abf5efc1987cb8f4351a4c9a60878.

**Comment by gmella** [comment]
I just add a look on `eXide / templates / existdb / controller.xql` that doesn't seem to be fixed in the same way. If you also fix this files, this will prevent all futur applications created on top of this template.
Hope this is clear...
